src/sidebars/sidebar-controller.js

```diff
@@ -51,8 +54,25 @@
                 this._showFavourites = true;
             }
         }
+
+        _showSearchesChanged() {
+            var showSearches = this.showSearches();
+            if (showSearches) {
+                //Couldn't get it to work reliable in a synchronous/promise fashion
+                setTimeout(function() {
```

Comment:
`setTimeout` shouldn't be used in angular apps. Instead inject `$timeout` and use that. Also would prefer the ES6 style function syntax.